tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -2034,6 +2034,45 @@ def test_max_invalid_shapes_and_axis(device, array, axis, dtype, is_module):
             a.max(axis)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    # Special shapes
+    chainer.testing.product({
+        'shape': [(), (0,), (1,), (2, 0, 3), (1, 1, 1), (2, 3)],
+        'in_dtypes,out_dtype': (
+            _make_same_in_out_dtypes(2, chainerx.testing.numeric_dtypes)),
```

Comment:
Could you add mixed-dtypes tests? (Sorry, I should have said when reviewing `TestMinimum`.)